lldb/test/API/python_api/interpreter/TestCommandInterpreterAPI.py

```diff
@@ -177,7 +177,8 @@ def test_get_transcript(self):
         # (lldb) r
         self.assertEqual(transcript[3]["command"], "r")
         self.assertEqual(transcript[3]["commandName"], "process launch")
-        self.assertEqual(transcript[3]["commandArguments"], "-X true --")
+        # Not checking `commandArguments`, because it can be different on different platforms/configurations.
+        # On macOS, it's "-X true --". On Linux, it's "-c/bin/bash --".
```

Comment:
Do you really need to launch a process for this test? Maybe just don't use "process launch"? Will save some time on the test bots not having to launch a process.

Reply:
We do need a target. At the end of this test, it validates that the output of "statistics dump" command is a valid JSON object. In order for such output to be generated, we need a target. Otherwise the command says:
> (lldb) statistics dump
> error: invalid target, create a target using the 'target create' command